update-server/config/config.php

```diff
@@ -88,8 +88,8 @@
 	],
 	'beta' => [
 		'10.4.0' => [
```

Comment:
Should this change to:
```suggestion
		'10.4.1' => [
```
?
@VicDeo

Reply:
no, it isn't.
https://github.com/owncloud/administration/blob/1753633c80ffeca8efc13beb14984fcf9d8746c6/update-server/config/config.php#L30-L40